### DISCUSSION TITLE
TypeError in browser console after update to 3.1.0

### DISCUSSION THREAD

**Discussion by ganeshkrishnasharma** [chakra-ui/chakra-ui#9111] [General]
After updating to v3.1.0, the following error repeatedly pops up in browser console. Please take a look

```
@chakra-ui_react.js?v=bcd550b0:39296 Uncaught TypeError: Cannot read properties of undefined (reading 'ownerDocument')
    at getDefaultParent (@chakra-ui_react.js?v=bcd550b0:39296:23)
    at hideOthers (@chakra-ui_react.js?v=bcd550b0:39401:40)
    at @chakra-ui_react.js?v=bcd550b0:39424:22
    at @chakra-ui_react.js?v=bcd550b0:39413:47
```

any help to resolve this is appreciated, thanks in advance

**Comment by ganeshkrishnasharma** [comment]
Hi @segunadebayo , please kindly have a look , this happens with v3.1.1 as well. ( typescript version - 5.6.2 )

**Comment by segunadebayo** [comment]
Closing as stale. Please start a new discussion if this is still relevant on the latest version.
